proxies/shared/partials/Partial.Flows.CreateMessageEndpoint.xml

```diff
@@ -14,9 +14,11 @@
         <Step>
             <Name>AssignMessage.Messages.Create.Request</Name>
         </Step>
-        <Step>
-            <Name>AssignMessage.AuthenticationDetails</Name>
-        </Step>
+        {% if ENVIRONMENT_TYPE != 'sandbox' %}
```

Comment:
@tdroza-nhs  / @aaron-downing  - any issues with me adding in this for the `/message` endpoint? ATM we cannot test the functionality of the Authorization headers in sandbox environments for this endpoint. 

For example if I make a POST request to `https://sandbox.api.service.nhs.uk/comms/v1/messages` with an Authorization header value of `banned` I would've expected to see a `banned` response rather than a successful 201 similar the `message-batches` endpoint.

Reply:
Looks good to me. Hopefully will eventually move to using the `Prefer` header for specific status codes we want to test instead of relying on other request headers/content

Reply:
Sounds like a really good idea!